pkg/registry/registry.go

```diff
@@ -1,4 +1,16 @@
-// Package registry implements a docker V2 registry.
+// Package registry implements a docker V2 registry and the OCI distribution specification.
+// 
+// It is designed to be used anywhere a low dependency container registry is needed, with an 
+// initial focus on tests. 
+//
+// It's goal is to be standards compliant and it's strictness will increase over time.
+//
+// As an example
```

Comment:
Pull this out into an [example test](https://blog.golang.org/examples)?

Reply:
Done